src/FixMath.h

```diff
@@ -512,12 +521,12 @@ class UFix
   /** The number of bits used to encode the integral part.
       @return The number of bits used to encode the integral part.
   */
-  int8_t getNI() const {return NI;}
+  static constexpr int8_t getNI() /*const*/ {return NI;}
```

Comment:
True, that's better. +1 for the constexpr, too. (Suggest removing the commented ```/*const*/```, it's implied by static).

Reply:
Ah, yes, that remained from some tests… Good thing is that now these functions can be used as template parameters :)